xDrip/Scenes/Home/Home/HomeViewController.swift

```diff
@@ -64,4 +68,19 @@ class HomeViewController: UIViewController, HomeDisplayLogic {
     func displayLoad(viewModel: Home.Load.ViewModel) {
         
     }
+    
+    private func setupUI() {
+        let titles = [NSLocalizedString("home_time_frame_1h", comment: "1H"),
+                      NSLocalizedString("home_time_frame_3h", comment: "3H"),
```

Comment:
Use `"home_time_frame_3h".localized`

Reply:
<img width="795" alt="image" src="https://user-images.githubusercontent.com/41580561/77305679-27e4fc80-6cff-11ea-9685-beeb783d593f.png">

Looks like something is missed.

Reply:
`import AKUtils`